OrbitClientProtos/preset.proto

```diff
@@ -14,3 +14,8 @@ message PresetInfo {
   string process_full_path = 1;
   map<string, PresetModule> path_to_module = 2;
 }
+
+message PresetFile {
```

Comment:
So maybe add a comment that this is not used for serialization.

Reply:
Yes, let's make it clear